charts/sps-sample/values.yaml

```
@@ -126,3 +161,150 @@
+    primary:
+      auth:
+        username: keycloak-user
+        password: admin
+        database: keycloak-db
+      podSecurityContext:
+        fsGroup: null
+      containerSecurityContext:
+        runAsUser: null
+        runAsNonRoot: true
+        allowPrivilegeEscalation: false
+        seccompProfile: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)